src/Datasource/Connection.php

```diff
@@ -34,39 +33,31 @@ class Connection implements ConnectionInterface
 {
     /**
      * Whether or not query logging is enabled.
-     *
-     * @var bool
```

Comment:
Isn't this required by cakephp cs?

Reply:
nope

Reply:
and I wouldn't see the need why we need it for Cake5 based plugins. Native property types are required anyway